src/components/pie-chart.js

```diff
@@ -122,41 +94,124 @@ function PieChart({ data, content }) {
+        <path d="m176.952 339.052h-32.175c-7.243 0-13.136-5.893-13.136-13.136s5.893-13.136 13.136-13.136h54.323v-30h-27.177v-20.239h-30v20.344c-22.458 1.476-40.282 20.206-40.282 43.031 0 23.785 19.351 43.136 43.136 43.136h32.175c7.871 0 14.274 6.404 14.274 14.275s-6.403 14.274-14.274 14.274h-69.333v30h34.304v20.239h30v-20.239h5.029c24.413 0 44.274-19.861 44.274-44.274s-19.861-44.275-44.274-44.275z" />
+      </g>
+    </svg>
+  )
+
+  return (
+    <CentralToTotalComparisonPie
+      data={parsedData}
+      content={parsedContent}
+      formatValue={formatToUSD}
+      metricIcon={icon}
+      colors={colors}
+    />
+  )
+}
+
+export const StaffPieChart = ({ data, content }) => {
+  const parsedData = {
+    centralPrograms: data["eoy_total_positions"],
+    allOUSD: 5000 /*data["all_ousd_eoy_total_positions"]*/,
+    change: data["change_from_previous_year"]["eoy_total_positions"],
+  }
+  const parsedContent = {
+    centralProgramsLabel: content["central_programs"],
+    otherLabel: content["other_staff"],
+    increase: content["increase"],
+    decrease: content["decrease"],
+    description: content["description_staff"],
+    statDescriptor: content["stat_descriptor_staff"],
+  }
+  const colors = {
+    [parsedContent.centralProgramsLabel]: "#129c9b",
+    [parsedContent.otherLabel]: "#60697d",
```

Comment:
How do you feel about the light gray from the theme?  `$light-gray: #e9e9e9;`

And making the label darker with something like...
```javascript
          radialLabelsTextColor={{
            from: "color",
            modifiers: [["darker", 2]],
          }}
```

<img width="455" alt="Screen Shot 2021-04-27 at 8 02 13 PM" src="https://user-images.githubusercontent.com/1191015/116340321-d7f32780-a793-11eb-8964-ae61fb5e52c5.png">

Reply:
I like it!

Reply:
I'll see if there's a way to pull directly from the them for the colors